src/SIM/SIMoutput.C

```diff
@@ -513,8 +530,11 @@ bool SIMoutput::writeGlvS (const Vector& psol, int iStep, int& nBlock,
 {
   idBlock = this->writeGlvS1(psol,iStep,nBlock,time,
                              pvecName,idBlock,psolComps);
+
   if (idBlock < 0)
     return false;
+  else if (!this->writeAddFuncs(iStep,nBlock,50,time))
```

Comment:
Careful with the idBlock value here (50). Ensure it does not conflict with others.

Reply:
50 is the default from writeGlvF, but i now make sure to increase it for each registered function (starting at 50).